src/core_ocean/analysis_members/Registry_eddy_product_variables.xml

```diff
@@ -39,12 +39,28 @@
 		<var name="velocityMeridionalTimesTemperature" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
 			description="cell-wise product of component of horizontal velocity in the northward direction and temperature"
 		/>
-                <var name="velocityZonalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
-                        description="cell-wise product of component of horizontal bolus velocity in the eastward direction and temperature"
-                />
-                <var name="velocityMeridionalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
-                        description="cell-wise product of component of horizontal bolus velocity in the northward direction and temperature"
-                />
+		<var name="normalVelocitySquared" type="real" dimensions="nVertLevels nEdges Time" units="m^2 s^{-2}"
+			description="edge based square of normal velocity"
+		/>
+		<var name="normalVelocityTimesTemperature" type="real" dimensions="nVertLevels nEdges Time" units="m s^{-1} C"
+			description="edge based product of normal velocity and temperature"
+		/>
+		<var name="velocityZonalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
+			description="cell-wise product of component of horizontal bolus velocity in the eastward direction and temperature"
+			 packages="gm"
```

Comment:
@luke, I added all the GM variables to the gm package, so they do not use memory space for RRS runs.